cylc/flow/scheduler_cli.py

```diff
@@ -309,12 +309,8 @@ def get_option_parser(add_std_opts: bool = False) -> COP:
         argdoc=[WORKFLOW_ID_ARG_DOC]
     )
 
-    options = parser.get_cylc_rose_options() + PLAY_OPTIONS
-    for option in options:
-        if isinstance(option, OptionSettings):
```

Comment:
There are no longer any options stored as Dicts rather than OptionSettings objects :. this is redundant.